docs/rules/DAP245.md

```diff
@@ -0,0 +1,17 @@
+﻿# DAP245
+
+Dapper.Vanilla does not handle pseudo-positional parameters in the string correctly. And since Dapper.AOT uses Dapper.Vanilla, it also has this issue.
```

Comment:
Dapper.Vanilla isn't a thing; rephrase somehow?

Reply:
Also, Dapper.AOT usually doesn't use Dapper.Vanilla

Reply:
Tried to change the wording - can you please take a look if that is formulated finely?